app/src/test/kotlin/fr/free/nrw/commons/media/MediaDetailFragmentUnitTests.kt

```diff
@@ -173,8 +176,10 @@ class MediaDetailFragmentUnitTests {
         view = LayoutInflater.from(activity)
             .inflate(R.layout.fragment_media_detail, null) as View
```

Comment:
The view binding should be replacing this view.  I think you can safely delete this and switch to looking at the fields in the binding instead of the `findViewById()` calls.